.github/workflows/build.yml

```diff
@@ -23,9 +23,7 @@ jobs:
       # INFO: Latest `devenv` version is not compatible with `nixos-24.11`
       # channel
       - name: Install devenv
-        run: |
-          nix profile install --accept-flake-config github:cachix/devenv/v1.4.1
-          devenv version
```

Comment:
@oscar-izval I'm reviewing the `devenv version` command from this block as it could be inferred from the tag.